package.json

```diff
@@ -1158,10 +1252,26 @@
           "when": "cmake:enableFullFeatureSet",
           "command": "cmake.ctest"
         },
+        {
+          "when": "cmake:enableFullFeatureSet && useCMakePresets",
```

Comment:
Just to keep the same style and pattern, please switch the order of these to have `command` first. It seems like there are some others that were already there like that, but we really should have `command` first in all of them.